src/query/ObjectSelector.js

```diff
@@ -33,15 +38,19 @@ export default class ObjectSelector {
     this.unhighlight();
     let attributes = {};
     if (!object) {
+      const slicerDataSource = this.viewer.dataSources.getByName('slicer')[0];
       const objects = this.scene.drillPick(clickPosition, DRILL_PICK_LIMIT, DRILL_PICK_LENGTH, DRILL_PICK_LENGTH);
       object = objects[0];
+      if (object && object.id && slicerDataSource.entities.contains(object.id)) {
+        object =
```

Comment:
Could you add a comment?
Could you also simplify next line by introduction an intermediary variable or using a plain "if" construct?